src/domain.c

```diff
@@ -760,4 +769,15 @@ domain_x509_verify(int ok, X509_STORE_CTX *ctx)
 
 	return (ok);
 }
+
+void
+kore_domain_reload_certificates(void)
+{
+	static struct kore_domain	*dom;
```

Comment:
why on earth is this stack variable declared static?

Reply:
done. (I'm trying to find which issue is outstanding, then I will rebase the patch on latest)